### PR TITLE
Fix hhs national from google symptoms

### PR DESCRIPTION
**Pull request by jingjtang**
### Description
This PR is used to fix the daily report for HHS and National level.

The count values in Google Symptoms are not exactly counts, but have been pre-scaled by Google already. Such pre-processing makes the "counts" that we get from Google are in an arbitrary unit. Previously we have a simple aggregated (summation of the counts from states) from state level report to HHS and National level report which is incorrect. Now, we fix those mapping to make them match what we did for the geo mapping from county to HRR and MSA. 

Notice that, as what we did for the geo mapping from county to MSA and HRR, the finalized `counts` for higher geographical levels should be pretty small since we treat counties with no report as 0 report. 

The previous [PR ](https://github.com/cmu-delphi/covidcast-indicators/pull/1418) was mistakenly closed after renaming the branch. 

### Changelog
Itemize code/test/documentation changes and files added/removed.
- `run.py` and `geo.py`
- `tests/test_geo.py`

### Fixes 
- Fixes [#1406](https://github.com/cmu-delphi/covidcast-indicators/issues/1406)

### CODE REVIEW

**Comment by jingjtang** [comment]
Record the discussion related to either generating the HHS/national level report from state level or county level reports

```
krivard 1 hour ago
Do we want to use the state data to generate hhs and nation?
  
Member
@ryantibs ryantibs 1 hour ago
Thanks for catching this! I was about to raise the same issue 🙂

And yes---I don't see why we would generate it from counties provided that we have states in-hand (and even aggregating up from counties --> states is not going to be ideal since they won't recover states).
  
Member
Author
@jingjtang jingjtang 1 hour ago • 
edited 
No. We already know that at least for the current Google Symptoms (Anosmia and Ageusia only), we have only around ~100 counties available. Since we do a population weighted average, it should be more accurate to do the aggregation from county level. For example, we only have 2 counties available in state A, if we start from state level, we will add a weight based on the state population to the already normalized and scaled count from Google when doing the average.
  
Member
Author
@jingjtang jingjtang 1 hour ago
Or do you think the other way is more reasonable?
  
Member
@krivard krivard 1 hour ago
we only have two counties available in state A, but we have no idea how many counties go into Google's value for state A, right? if they're able to include more counties in their published state figures, shouldn't we make use of them?
 
👍
1
  
Member
Author
@jingjtang jingjtang 1 hour ago
Yeah that makes sense, I may make too far assumption about this. Since a lot of things are unknown to us, I think you and Ryan is correct. I will change those mappings from state level.
  
Member
@ryantibs ryantibs 44 minutes ago
I can see there are pros/cons either way, but I do think it's worth aggregating state signals to build national and HHS signals, for ultimately the following reason: state numbers are based on normalizing state counts, and there's a lot less censorship here than at the county level (where lots of counts are censored), so you'll be able to leverage a lot more data this way.
```

**Comment by jingjtang** [comment]
Things to do for this fixing are:

[done by Sam] Change the visualization strategy for Google Symptoms. We need to carefully label that there should be no comparison between regions at the same geographical level and also between geographical levels.
[waiting for review] Fix the aggregation method (from simple summation to population-weighted average) for geo mapping from county to HHS and Nation.

File: google_symptoms/delphi_google_symptoms/geo.py [hunk 1]
```diff
@@ -23,20 +23,30 @@ def generate_transition_matrix(geo_res):
...
+    if geo_res in ["hrr", "msa"]:
+        mapping_flag = "fips"
+        map_df = gmpr.get_crosswalk("fips", geo_res)
+        # Add population as weights
+        map_df = gmpr.add_population_column(map_df, "fips")
+    else:
+        mapping_flag = "state_id"
+        map_df = gmpr.get_crosswalk("state", "state")
+        map_df = gmpr.add_geocode(map_df, "state_code", "hhs")
+        map_df = gmpr.add_geocode(map_df, "state_code", "nation")
```
Comment (krivard) [inline google_symptoms/delphi_google_symptoms/geo.py:35]: Would this work?
```suggestion
        map_df = gmpr.add_geocode(map_df, "state_code", geo_res)
```

**Review by krivard — APPROVED** [review]
This is tough to test since it depends on GeoMapper populations, but from what I can see it extends the HRR/MSA implementation but otherwise exactly matches it, which is what we wanted.

**Pull request merged**